packages/flutter/lib/src/foundation/assertions.dart

```diff
@@ -867,7 +867,8 @@ class FlutterError extends Error with DiagnosticableTreeMixin implements Asserti
   /// The default behavior is to call [presentError].
   ///
   /// You can set this to your own function to override this default behavior.
-  /// For example, you could report all errors to your server.
+  /// For example, you could report all errors to your server. It's recommended
+  /// to call the [presentError] to be able to see the logs in the console.
```

Comment:
What's the reason to not recommend it?

Reply:
Is there a way we can make presentError be called no matter what if structured errors are supposed to be enabled (default behavior)? Something like:

```
if user specified onError handler
 onError = () {
   presentError();
   onError();
 }
```

Reply:
I've considered that, but.. how do we avoid duplication in case structured errors enabled, then disabled, and then enabled here?https://github.com/flutter/flutter/blob/d2b5e312a2e2f3cfd6dc6ea070cdf7335e16586f/packages/flutter/lib/src/widgets/widget_inspector.dart#L996-L1003

Or what if in onError they already call `presentError`? Or how do we event disabled it then ...

Reply:
And even with that approach, after the errors are enabled on service connection, developer still might set onError to some callback and it will stop working

Reply:
cc @jacob314 if you have any opinion on structured errors